packages/ui/Input/deprecated/Select.tsx

```diff
@@ -0,0 +1,139 @@
+import { useState, useRef, useEffect, useCallback } from 'react';
+import { IconChevronDown, IconUser } from '@sopt-makers/icons';
+import * as S from '../style.css';
+
+export interface Option<T> {
+  label: string;
+  value: T;
+  description?: string;
+  icon?: React.ReactNode;
+  profileUrl?: string;
+}
+
+interface SelectProps<T> {
+  className?: string;
+  placeholder?: string;
+  type: 'text' | 'textDesc' | 'textIcon' | 'userList' | 'userListDesc';
+  options: Option<T>[];
+  visibleOptions?: number;
+  defaultValue?: Option<T>;
+  onChange: (value: T) => void;
+}
+
+/**
+ * Select 컴포넌트는 더 이상 권장되지 않지만 계속 사용할 수 있습니다.
+ * 새로운 `SelectV2` 컴포넌트 사용을 권장합니다.
+ */
```

Comment:
```suggestion
/**
 * @deprecated
 * Select 컴포넌트는 더 이상 권장되지 않지만 계속 사용할 수 있습니다.
 * 새로운 `SelectV2` 컴포넌트 사용을 권장합니다.
 */
```
<img width="600" alt="image" src="https://github.com/user-attachments/assets/9b78c8f9-e648-4874-aea0-fcf2eb17a50d">
`@deprecated` 태그를 사용하면 더 명시적으로 해당 컴포넌트가 deprecated 됨을 알릴 수 있어요!

Reply:
> <img alt="image" width="600" src="https://private-user-images.githubusercontent.com/97586683/369957524-9b78c8f9-e648-4874-aea0-fcf2eb17a50d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNTcwNDIsIm5iZiI6MTcyNzE1Njc0MiwicGF0aCI6Ii85NzU4NjY4My8zNjk5NTc1MjQtOWI3OGM4ZjktZTY0OC00ODc0LWFlYTAtZmNmMmViMTdhNTBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDA1NDU0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVlZjFhYzMyZTE1YmFkNDc0NTZmNDUwYzJlNGY3NTI0ZjcxYWU2MThlNmUwMTMwNDU5NjM2YWI4NTMyZDFkN2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.RXJKLMr4LGKIgmqtK0MaCqlpqW_c011-LYP9kT8FsYw">
> `@deprecated` 태그를 사용하면 더 명시적으로 해당 컴포넌트가 deprecated 됨을 알릴 수 있어요!

와 이거 좋네요.......👍🏻